guides/source/caching_with_rails.md

```diff
@@ -370,6 +370,32 @@ There are some common options that can be used by all cache implementations. The
 
 * `:race_condition_ttl` - This option is used in conjunction with the `:expires_in` option. It will prevent race conditions when cache entries expire by preventing multiple processes from simultaneously regenerating the same entry (also known as the dog pile effect). This option sets the number of seconds that an expired entry can be reused while a new value is being regenerated. It's a good practice to set this value if you use the `:expires_in` option.
 
+#### Connection Pool Options
+
+By default the `MemCacheStore` and `RedisCacheStore` use a single connection
+per process. This means that if you're using Puma, or another threaded server,
+you can have multiple threads waiting for the connection to become available.
+To increase the amount of available connections you can enable connection
+pooling. Connection pooling is only supported by the `MemCacheStore` and
+`RedisCacheStore`.
+
+To enable the connection pool first add the `connection_pool` gem to your
```

Comment:
The preceding paragraph now mentions that we're enabling connection pooling, so I think we can shorten this:

```suggestion
First, add the `connection_pool` gem to your Gemfile:
```

Reply:
I've applied the suggested changes.